src/BPEssentials/FileChecker.cs

```diff
@@ -7,6 +7,13 @@ namespace BPEssentials
 {
     public static class FileChecker
     {
+        private static readonly Dictionary<string, string> RequiredDirectorys = new Dictionary<string, string>
```

Comment:
Make this a getter only property.
```suggestion
        public static Dictionary<string, string> RequiredDirectorys { get; }= new Dictionary<string, string>
```